d2bs/kolbot/libs/common/Config.js

```diff
@@ -161,6 +161,22 @@ var Config = {
 		[0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
 		[0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
 	],
+	SortInventory: true,

```

Comment:
Overall change looks good, should the default be to use sorting? (Same question for the individual char config files)

Reply:
It's a good question. I have noticed that it can cause about a 1-2 sec wait during town visits as the character assesses whether anything can be sorted. So turning it off by default would make town visits slightly faster.